### PR TITLE
avoid GET parameters being overrided in CategoryAdmin

### PR DESCRIPTION
**Pull request by dem3trio**
The ``getPersistentParameters()`` inside the ``CategoryAdmin`` overrides all GET parameters.

I've been using the SonataTranslationBundle with the ClassificationBundle. The TranslationBundle needs to use an specific parameter (``tl=yourLocale``) and the ``getPersistentParameters()`` function doesn't call the ``parent::getPersistentParameters()`` so the parameter disappears.

### CODE REVIEW

**Comment by OskarStark** [comment]
Can you please check travis?

**Comment by dem3trio** [comment]
It seems that the problem has been inherited from the master branch.

this is my execution: https://travis-ci.org/sonata-project/SonataClassificationBundle/builds/82127080 
and this is the latest from master: https://travis-ci.org/sonata-project/SonataClassificationBundle/builds/81801617

The problem is in SonataClassificationBundle/Controller/Api/CategoryController.php and I have only changed SonataClassificationBundle/Admin/CategoryAdmin.php

**Comment by Soullivaneuh** [comment]
@dem3trio tests fixed in c40110d, please rebase your PR.

**Comment by dem3trio** [comment]
@Soullivaneuh I have make the rebase to c40110d but the test are failing because of a [timeout](https://travis-ci.org/sonata-project/SonataClassificationBundle/jobs/82574519) just the same of the [master](https://travis-ci.org/sonata-project/SonataClassificationBundle/builds/82156421)

**Comment by OskarStark** [comment]
@dem3trio  can you please make a new push force? to trigger travis?

File: Admin/CategoryAdmin.php [hunk 1]
```diff
@@ -164,6 +164,8 @@ public function getPersistentParameters()
             'hide_context' => $this->hasRequest() ? (int) $this->getRequest()->get('hide_context', 0) : 0,
         );
 
+        $parameters = array_merge( parent::getPersistentParameters(), $parameters);
```
Comment (Soullivaneuh) [inline Admin/CategoryAdmin.php]: `array_merge( parent` => `array_merge(parent`

Comment (OskarStark) [inline Admin/CategoryAdmin.php]: ping @dem3trio

**Comment by OskarStark** [comment]
@dem3trio can you please execute ``make cs`` to fix travis?

```
Legend: ?-unknown, I-invalid file syntax, file ignored, .-no changes, F-fixed, E-error
   1) Admin/CategoryAdmin.php (parenthesis)
```

Thank you

**Comment by dem3trio** [comment]
Done. It was a white space (that I added before to force travis in other push)

**Comment by dem3trio** [comment]
Timeout again:

Timeout (20 minutes) reached. Terminating "composer update --prefer-dist --no-interaction --prefer-lowest"

how can i make a force push without change the code?

I've tried with a git push origin fix_parameters --force but it doesn't works

**Comment by core23** [comment]
Please rebase @dem3trio

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**